File: app/src/main/java/com/BlizzardArmory/diablo/character/Skills.java
package com.BlizzardArmory.diablo.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang3.builder.ToStringBuilder;

import java.util.List;

public class Skills {

    @SerializedName("active")
    @Expose
    private List<Active> active = null;
    @SerializedName("passive")
    @Expose
    private List<Passive> passive = null;

    public List<Active> getActive() {
        return active;
    }

    public void setActive(List<Active> active) {
        this.active = active;
    }

    public List<Passive> getPassive() {
        return passive;
    }

    public void setPassive(List<Passive> passive) {
        this.passive = passive;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("active", active).append("passive", passive).toString();
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/character/skills/Skills.kt
package com.BlizzardArmory.model.diablo.character.skills

import com.google.gson.annotations.SerializedName


/**
 * The type Skills.
 */
data class Skills(

    @SerializedName("active")
    var active: List<Active>,

    @SerializedName("passive")
    var passive: List<Passive>

)